src/test/librbd/test_librbd.cc

```diff
@@ -19,6 +19,7 @@
 #include "include/rbd/librbd.hpp"
 #include "include/event_type.h"
 #include "include/err.h"
+#include "common/ceph_mutex.h"
```

Comment:
might want to `s/ mutex / std::mutex /` in this file to avoid the ambiguities
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/librbd/test_librbd.cc: In member function 'virtual void TestLibRBD_UpdateWatchAndResize_Test::TestBody()':
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/librbd/test_librbd.cc:714:5: error: reference to 'mutex' is ambiguous
  714 |     mutex m_lock;
      |     ^~~~~
```